circuits/cpp/barretenberg/py/src/estimation/main.py

```diff
@@ -0,0 +1,71 @@
+from flavors import GoblinUltra
+from circuit import CircuitKZG
+from goblin import Goblin
+from protogalaxy import FoldingVerifier, Decider
+from utils import DataLog
+
+# App circuits an kernel circuits are in a "stack" and all
+# are assumed to have a fixed size
+LOG_N_IN_STACK = 13
+
+# A Goblin instance conists of three circuit instances, one for each proof output
```

Comment:
Succinct description of the data model and what we're trying to do with this tool